blazar_dashboard/content/leases/views.py

```diff
@@ -68,6 +68,16 @@ def calendar_data_view(request, resource_type):
     resources, reservations = api_mapping[resource_type](request)
     data['resources'] = resources
     data['reservations'] = reservations
+    # Include authorized_projects capability for devices in response
+    if resource_type == "device":
+        data["project_id"] = request.user.project_id
+        for device in resources:
+            full_device = api.client.device_with_capabilities(request, device["id"])
```

Comment:
This turns this into an O(N) operation and we pay overhead on these requests. Do the capabilities not come back on the device list request? Is it possible to include them on the device list request via some flag, if they are not returned by default?